src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -3324,6 +3318,8 @@ public FormValidation doCheckLocal(@QueryParameter String value) throws IOExcept
      * Enables trace logging of Ganymed SSH library.
      * <p>
      * Intended to be invoked from Groovy console.
+     * @deprecated
```

Comment:
I  would even remove this (and so dependency on trilead-api) as the class in trilead-api is even already deprecated. 
https://github.com/jenkinsci/trilead-ssh2/blob/d9e5b647ca51e65db86e29a642f85eaee560bd10/src/com/trilead/ssh2/DebugLogger.java#L11